guides/release/configuring-ember/debugging.md

```diff
@@ -68,6 +68,9 @@ export default Application.extend({
 ```
 ### Dealing with deprecations
 
+In addition to what is described in the [Handling Deprecations guide](../handling-deprecations),
```

Comment:
```suggestion
In addition to what is described in the [Handling Deprecations guide](../configuring-ember/handling-deprecations),
```

Gave this error: https://travis-ci.org/github/ember-learn/guides-source/builds/682150293#L6548-L6560

Saw it working here: https://github.com/ember-learn/guides-source/blob/master/guides/release/upgrading/index.md

Reply:
Hm, this doesn't work locally because `handling-deprecations.md` and `debugging.md` are siblings and are both inside `configuring-ember`. This might be that issue of the link checker not handling nested links properly again?